README.md

```diff
@@ -1,6 +1,20 @@
 ZMTP Wireshark Dissector
 ========================
 
+*THIS LUA DISSECTOR HAS BEEN OBSOLETED!!!!*
+
+*A built-in dissector, written in C, has been merged in April 2024, see
+https://gitlab.com/wireshark/wireshark/-/merge_requests/14947*
+
+*This dissector has been renamed to "zmtp-lua" (and the dissector table has been renamed to "zmtp-lua.protocol"),
+to allow this dissector to load without clashing names with the C version.
+Existing port registrations should continue to work
+(where you can register for the table for either version),
+you can also still do decode-as for either protocol.  The built-in version additionally has a table in the
+preferences that will survive restarts.  Please do report on any regressions seen from this implementation -
```

Comment:
Report where?

Reply:
https://gitlab.com/wireshark/wireshark/-/issues, or ask.wireshark.org, or the the mailing lists.  Have updated the change with these details.